code/game/objects/items/weapons/storage/pouches.dm

```diff
@@ -322,20 +323,60 @@
 		/obj/item/gun/projectile/silenced,
 		/obj/item/gun/projectile/ladon,
 		/obj/item/gun/projectile/glock,
+		/obj/item/gun/projectile/that_gun,
+		/obj/item/gun/projectile/judiciary,
+		/obj/item/gun/projectile/rebar,
+		/obj/item/gun/projectile/rivet,
+		/obj/item/gun/projectile/silvereye,
+		/obj/item/gun/projectile/spring,
+		/obj/item/gun/projectile/boltgun/sawn,
+		/obj/item/gun/projectile/shotgun/pump/obrez,
+		/obj/item/gun/projectile/automatic/texan,
+		/obj/item/gun/projectile/automatic/sts/sawn,
+		/obj/item/gun/projectile/automatic/luty,
+		/obj/item/gun/projectile/automatic/freedom,
+		/obj/item/gun/projectile/automatic/slaught_o_matic,
+		/obj/item/gun/projectile/automatic/nordwind/strelki/sawn,
+		/obj/item/gun/projectile/automatic/omnirifle/solmarine/sawn,
+		/obj/item/gun/projectile/automatic/omnirifle/solmarine/shotgunless_sawn,
 		/obj/item/gun/energy/glock,
 		/obj/item/gun/energy/ionpistol,
 		/obj/item/gun/energy/gun,
 		/obj/item/gun/energy/chameleon,
-		//obj/item/gun/energy/captain, //too unwieldy, use belt/suit slot or other storage
+		/obj/item/gun/energy/centurio,
 		/obj/item/gun/energy/stunrevolver,
+		/obj/item/gun/energy/ntpistol,
+		/obj/item/gun/energy/zwang,
+		/obj/item/gun/energy/captain/zapper,
+		/obj/item/gun/energy/floragun,
+		/obj/item/gun/energy/taser,
+		/obj/item/gun/energy/slimegun,
+		/obj/item/gun/energy/sonic_emitter,
+		/obj/item/gun/energy/crossbow,
+		/obj/item/gun/energy/xray/psychic_cannon,
+		/obj/item/gun/energy/laser/railgun/pistol,
+		/obj/item/gun/energy/laser/makeshift,
+		/obj/item/gun/energy/cog/sawn,
 		/obj/item/gun/projectile/revolver,
+		/obj/item/gun/projectile/revolver/sixshot/sawn,
 		/obj/item/gun/projectile/shotgun/doublebarrel/sawn, //short enough to fit in
 		/obj/item/gun/launcher/syringe,
 		/obj/item/gun/energy/plasma/auretian,
-		/obj/item/gun/energy/centurio,
-		/obj/item/gun/projectile/boltgun/flare_gun
```

Comment:
Why did you remove flare guns from the list?

Reply:
I didnt its moved down

Reply:
oh my bad